app/src/main/java/com/elavon/converge/TransactionService.java

```diff
@@ -185,6 +188,11 @@ public void getBalanceInquiry(
             transactionManager.processBalanceInquiry(balanceInquiry, requestId, listener);
         }
 
+        @Override
+        public void captureAllTransactionsWithOptions(String s, Bundle bundle, CaptureAllRequest captureAllRequest, IPoyntTransactionCaptureAllListener iPoyntTransactionCaptureAllListener) throws RemoteException {
```

Comment:
use indentation, use proper naming instead of `String s`


```
public void captureAllTransactionsWithOptions(String s, 
                                             Bundle bundle, 
                                             CaptureAllRequest captureAllRequest, 
                                             IPoyntTransactionCaptureAllListener iPoyntTransactionCaptureAllListener) 
                                             throws RemoteException 
```